components/StarshipList/index.js

```diff
@@ -0,0 +1,19 @@
+import useSWR from "swr";
+import Flex from "../Layout/Flex";
+
+import StarshipCard from "../StarshipCard";
+
+export default function StarshipList() {
+  const { data: starships, isLoading } = useSWR("/api/starships/");
+  if (!starships || isLoading) {
+    return <h1>Loading...</h1>;
+  }
+
+  return (
+    <Flex gap="26px" flexWrap="wrap" justifyContent="center">
```

Comment:
I would avoid "magic" pixel values, and express everything in relationship of the root document text size (rem), like `<Flex gap="1.5rem"...>`.

This way you can create a "scale" of spacings, like 1rem, 2rem, 3rem...and the ratios between the values become easier to understand, like:

```js
const container = styled.div`
  display: flex;
  gap: 2rem;
  padding: 1rem;
`;
```

I read it as "set the padding of the component as the _double_ of the gap between its elements